BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -97,6 +79,7 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       navigationController?.setNavigationBarHidden(true, animated: animated)
     }
     reloadUIState()
+    startNotificationObserver()
```

Comment:
Is there a reason this can't be done in `init`?

Reply:
Added in viewdidLoad